action.yml

```diff
@@ -7,3 +7,6 @@ runs:
 branding:
   icon: 'shield'
   color: 'green'
+inputs:
+  types:
```

Comment:
Conventional commits calls these "types", therefore I'd go for the same terminology.